### ISSUE TITLE
Add command to update mon configmap for  disaster recovery

### ISSUE THREAD

**Issue by subhamkrai** [rook/kubectl-rook-ceph#19]
Add command for:

- [ ] `kubectl rook-ceph set-mon-endpoints`: For disaster recovery, update the `rook-ceph-mon-endpoints` configmap and restart the operator

**Comment by subhamkrai** [comment]
@travisn I think your pr #61  covers some part of this?

**Comment by travisn** [comment]
Yes, I'd suggest we close this with #61. The `set-mon-endpoints` could be done separately, but let's hold off on that for now.
